app/javascript/controllers/style_labels_controller.js

```diff
@@ -0,0 +1,19 @@
+import { Controller } from "@hotwired/stimulus"
+
+// Connects to data-controller="style-labels"
+export default class extends Controller {
+  static targets = [ "checkBox", "label" ]
```

Comment:
Remove spaces in brackets: https://github.com/airbnb/javascript?tab=readme-ov-file#whitespace--in-brackets